zcutil/build.sh

```diff
@@ -50,6 +50,9 @@ $0 [ --enable-lcov || --disable-tests ] [ --disable-mining ] [ --disable-rust ]
   If --enable-proton is passed, Zcash is configured to build the Apache Qpid Proton
   library required for AMQP support. This library is not built by default.
   It must be passed after the test/mining/Rust arguments, if present.
+
+  If --disable-libs is passed, Zcash is configured to not build any libraries like
```

Comment:
@radix42 does it makes sense for use to use libzcashconsensus, or have it as a documented option?